src/NosCore.GameObject/Networking/ClientSession/ClientSession.cs

```diff
@@ -261,7 +265,7 @@ public void ChangeMapInstance(Guid mapInstanceId, int? mapX, int? mapY)
                 }
 
                 Character.MapInstance.LastUnregister = SystemTime.Now();
-                LeaveMap();
+                await LeaveMap();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)